tests/foreman/api/test_repository.py

```diff
@@ -103,6 +115,56 @@ def test_positive_create_with_name(self):
                     product=self.product, name=name).create()
                 self.assertEqual(name, repo.name)
 
+    @tier2
+    @upgrade
+    def test_positive_assign_http_proxy_to_repository(self):
+        """Assign http_proxy to Repositories.
+
+        :id: 5b3b992e-02d3-4b16-95ed-21f1588c7741
+
+        :expectedresults: HTTP Proxy can be assigned to repository.
```

Comment:
Please update the sync part as well here in doc string^^

Reply:
Updated PR